mlflow/config/__init__.py

```diff
@@ -0,0 +1,7 @@
+from mlflow.environment_variables import MLFLOW_ENABLE_SYNCHRONOUS_LOGGING
+
+
+def log_synchronously(enable=True):
+    """Enable or disable synchronous logging globally."""
+
+    MLFLOW_ENABLE_SYNCHRONOUS_LOGGING.set(enable)
```

Comment:
Can we make sure that `mlflow.config` shows up in our docs by adding an automodule sphinx entry similar to other entries in https://github.com/mlflow/mlflow/tree/master/docs/source/python_api?